app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class UserDisabled < StandardError
 
 class User < ActiveRecord::Base
   rolify
+  has_many :physical_tickets, through: :ticket_purchases, dependent: :destroy
```

Comment:
I guess that we are not going to use this relationship without scoping it by conference... so, better if you do something like this:

```ruby
has_many :physical_tickets, -> (conference) { where(conference: conference) }, through: :ticket_purchases, dependent: :destroy
```

If you do so, then the **tickets** method that you created (line 104 here) is not needed.